cron.go

```diff
@@ -142,6 +142,22 @@ func (c *Cron) AddFunc(spec string, cmd func()) (EntryID, error) {
 	return c.AddJob(spec, FuncJob(cmd))
 }
 
+// AddFuncCustomId add a cron with a custom ID
+func (c *Cron) AddFuncCustomId(id EntryID, spec string, cmd func()) error {
```

Comment:
what if the provided 'id' is not unique?
you need to vet the 'id' and make sure that it is unique.

Reply:
This translates to a lack of test coverage.
Please include tests to check expected functionality when 'id's are not unique (two jobs having the same 'id').

Reply:
The same as above. If this method is used, the external ID manager should be used to ensure the global uniqueness